backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxListenerApi.scala

```diff
@@ -68,13 +68,28 @@ class VeloxListenerApi extends ListenerApi with Logging {
           s"${COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED.key} should be enabled together.")
     }
 
+    if (
+      conf.get(COLUMNAR_VELOX_CACHE_ENABLED) &&
+      !conf.get(GLUTEN_SOFT_AFFINITY_ENABLED)
```

Comment:
GLUTEN_SOFT_AFFINITY_ENABLED is not defined in backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala.